Tiago:
  max_laser_value: 25.0 #25.0 # Value considered Ok, no wall
  min_laser_value: 0.05 # Value considered there is an obstacle or crashed
  new_ranges: 1 # How many laser scan reading considered (default dimesnion of tiago io 666)
  min_range: 0.3 # Minimum meters below wich we consider we have crashed
  min_linear_velocity : 0.0
  max_linear_velocity : 0.6
  min_angular_velocity : -0.5
  max_angular_velocity : 0.5
  init_linear_forward_speed: 0.0 # Initial linear speed in which we start each episode
  init_linear_turn_speed: 0.0 # Initial angular speed in shich we start each episode
  initial_angle : -1.91986
  angle_increment : 0.005774
  remove_scan : 33 # Number of scan to remove at the beginning and end of the laser scan this is euqalse to remove 20 degrees from FOV of robot then robot has 200 degrees 
  #18 # n scans remove because the first and last 20 scans of tiago are incorrect because the laser scan find the chassi of the robot
